pkg/stanza/operator/input/windows/input.go

```diff
@@ -71,7 +71,7 @@ func (i *Input) defaultStartRemoteSession() error {
 
 	sessionHandle, err := evtOpenSession(EvtRpcLoginClass, &login, 0, 0)
 	if err != nil {
-		return fmt.Errorf("failed to open session for server %s: %w", i.remote.Server, err)
+		return fmt.Errorf("failed to start remote session for server %s: %w", i.remote.Server, err)
```

Comment:
This can still be "open session" to differentiate from the error generated on line 111

Reply:
updated